### PR TITLE
github-298: facet addable multiple times

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Should take care of the issue: https://github.com/netarchivesuite/solrwayback/issues/298

### CODE REVIEW

File: src/js/src/components/SearchFacetOptions.vue [hunk 1]
```diff
@@ -78,10 +78,16 @@ export default {
...
     applyFacet(facetCategory, facet) {
+      let facetAllreadyApplied = false
       let newFacet = '&fq=' + facetCategory + ':"' + facet + '"'
-      this.updateSolrSettingOffset(0)
-      this.addToSearchAppliedFacets(newFacet)
-      this.$_pushSearchHistory('Search', this.query, this.searchAppliedFacets, this.solrSettings)
+      this.searchAppliedFacets.forEach((facet) => {
+        facet === newFacet ? facetAllreadyApplied = true : null
+      })
+      if(!facetAllreadyApplied === true) {
```
Comment (jorntx) [inline src/js/src/components/SearchFacetOptions.vue:86]: Could shorten this to `if(!facetAllreadyApplied) {`

**Review by jorntx — APPROVED** [review]
Looks good and works as well. Had one small comment which doesn't change a whole lot:)

I like your branch naming and I think we should adobt this approach going forward and maybe throw in an issue number.

File: src/js/src/components/SearchFacetOptions.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/js/src/components/SearchFacetOptions.vue]: Aaah! Yes, i'll fix that!
